cmd/tools/cli/apply.go

```diff
@@ -11,21 +11,56 @@ import (
 var applyCmd = &cobra.Command{
 	Use:   "apply <plugin_name>",
 	Short: "Apply the plugin",
-	Args:  cobra.ExactArgs(1),
+	//	Args:  cobra.ExactArgs(1),
```

Comment:
I think we can still keep this restriction as there would either one plugin or group at a time